src/Requests.php

```diff
@@ -842,6 +842,11 @@ public static function flatten($dictionary) {
 	 * @return string Decompressed string
 	 */
 	public static function decompress($data) {
+		if (trim($data) === '') {
+			// Not actually compressed.
```

Comment:
This comment seems misleading. If the `trim()` is empty, it means the body of the response was empty. While it is true it that `$data` is _not_ compressed, this makes it seem like we have an uncompressed body with instead, implying there _is_ `$data` available.

```suggestion
			// Empty body does not need further processing.
```